cost/google/instances_stackdriver_utilization/README.md

```diff
@@ -36,6 +35,7 @@ Provider tag value to match this policy: `gce`
 Required permissions in the provider:
 
 - The `Monitoring Viewer` Role
+- Scope for the credential is <https://www.googleapis.com/auth/monitoring.write https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/monitoring.read https://www.googleapis.com/auth/monitoring https://www.googleapis.com/auth/compute https://www.googleapis.com/auth/compute.readonly https://www.googleapis.com/auth/devstorage.full_control>
```

Comment:
scope is no longer needed.

Reply:
I have addressed this comment please review the changes.